### PR TITLE
Julia business site week 3

### PR DESCRIPTION
**Pull request by JuliaNiki**
https://koffie-business-site.netlify.app 

I am really proud of a hero video and how the website looks like with the first touches of Javascript. For this project I used Flexbox, CSS Entities information for accordion icon, made my own logo and found out more about ::after selector. 
The biggest problem I had was the hero video and its responsiveness on mobile. I found out about playsinline for the video tag to be able to start the video on mobile version.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,162 @@
...
+      <video preload="auto" autoplay muted loop playsinline>
+        <source src="./hero-seeds-video.mp4">
+        <p>Your browser does not support the video tag.</p>
+      </video>
+      <nav class="navigation-container">
+        <a class="navigation-tabs">about us</a>
+        <a class="navigation-tabs">shop</a>
+        <a class="navigation-tabs">learn</a>
+        <a class="navigation-tabs">contact us</a>
+        <img class="shopping-card" src="https://image.flaticon.com/icons/png/512/5498/5498014.png"
```
Comment (themisk84) [inline code/index.html:27]: Nice touch!

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,162 @@
...
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<body>
+  <!-- video or image as a header is cool :) -->
+  <header-container>
+    <header id="hero">
+      <video preload="auto" autoplay muted loop playsinline>
+        <source src="./hero-seeds-video.mp4">
+        <p>Your browser does not support the video tag.</p>
```
Comment (themisk84) [inline code/index.html:20]: Is this a message that it will be shown in case that the video can't be played right?Nice!

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,347 @@
...
+  position: absolute;
+  top: 0;
+  left:0;
+  z-index: 5;
+  object-fit: cover;
+}
+
+#hero .caption {
+  position: relative;
+  z-index: 20;
```
Comment (themisk84) [inline code/style.css:29]: Is there a reason that you put such a big value for the z-index?

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,347 @@
...
+  text-transform: uppercase;
+  font-size: 20px;
+  font-family: 'Varela Round', sans-serif;
+  margin-bottom: 20px;
+}
+#hero .caption h2 {
+  font-family: 'Varela Round', sans-serif;
+  font-size: 30px;
+}
+/* navigation-bar for mobile version is hidden */
```
Comment (themisk84) [inline code/style.css:44]: It helps the reviewer :)

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,347 @@
...
+.navigation-container {
+  position: absolute;
+  display: none;
+  font-size: 20px;
+  top: 8px;
+  right: 8px;
+  cursor: pointer;
+  z-index: 20;
```
Comment (themisk84) [inline code/style.css:52]: Same question as earlier about z-index value.

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,347 @@
...
+  padding: 17px;
+  text-transform: uppercase;
+  cursor: pointer;
+  color:white;
+  background-color:#271916;
+  border-radius: 20px;
+  border: 1px solid beige;
+}
+
+/* I made logo by myself this time */
```
Comment (themisk84) [inline code/style.css:75]: That is a well-designed logo!

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,347 @@
...
+  background: antiquewhite;
+}
+
+input[type="checkbox"] {
+  filter: invert(80%);
+}
+
+input[type="radio"] {
+  filter: invert(80%);
+}
```
Comment (themisk84) [inline code/style.css:154]: Cool!

File: code/style.css [hunk 8]
```diff
@@ -1,7 +1,347 @@
...
+  border-top: 0.5px solid #a38e8a;
+  border-radius: 5px ;
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
+.accordion-item-header.active + .accordion-item-body {
```
Comment (themisk84) [inline code/style.css:234]: Wouldn't it work if it was only .active + .accordion-item-body ?

**Review by themisk84 — APPROVED** [review]
Well done Julia! You created a wonderful page using many different html form elements and input types. Your code is clean written and your comments help the reviewer to read your code more easily. The styling is very nice and the page is responsive to different media queries. Keep u the good work!

File: README.md [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was made on third week for Technigo bootcamp.
+The aim of this week was to create a responsive business site with a hero image/video, signup form that would post data to a HTTPBin service and an accordion (FAQ) that can collapse to show and hide text. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I am really proud of a hero video and how the website looks like with the first touches of Javascript. For this project I used Flexbox, CSS Entities information for accordion icon, made my own logo and found out more about ::after selector.
```
Comment (camekman) [inline README.md:8]: Very nice description when you include what techniques you used!

File: code/index.html [hunk 10]
```diff
@@ -1,30 +1,162 @@
...
+      </video>
+      <nav class="navigation-container">
+        <a class="navigation-tabs">about us</a>
+        <a class="navigation-tabs">shop</a>
+        <a class="navigation-tabs">learn</a>
+        <a class="navigation-tabs">contact us</a>
+        <img class="shopping-card" src="https://image.flaticon.com/icons/png/512/5498/5498014.png"
+          alt="shopping-cart" />
```
Comment (camekman) [inline code/index.html:28]: Very nice with the shopping-cart icon added!

File: code/index.html [hunk 11]
```diff
@@ -1,30 +1,162 @@
...
-</html>
+  </div>
+  <footer>
+    <a target="_blank" href="https://github.com/JuliaNiki">
+      <p class=" footer"> 🐢 Github JuliaNiki ☕️ </p>
+    </a>
+  </footer>
+  <script src="./script.js"></script>
+  <!-- dont forget to link the javascript file here! -->
+</body>
```
Comment (camekman) [inline code/index.html:160]: You have a great way of structuring your code with a nice flow that makes it easy to read and understand. Great job!

File: code/script.js [hunk 12]
```diff
@@ -1,9 +1,21 @@
...
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById('question1').onclick = toggleFunction
+document.getElementById('question2').onclick = toggleFunction
+document.getElementById('question3').onclick = toggleFunction
+document.getElementById('question4').onclick = toggleFunction
+document.getElementById('question5').onclick = toggleFunction
+
+// Another way of doing accordion that I found just to remember myself  //
```
Comment (camekman) [inline code/script.js:15]: Great way of explaining what you have written in the js!

File: code/style.css [hunk 13]
```diff
@@ -1,7 +1,347 @@
...
+  top: 8px;
+  right: 8px;
+  cursor: pointer;
+  z-index: 20;
+  color: white;
+}
+.navigation-tabs {
+  margin: 20px;
+}
+/* shopping cart is also hidden for mobile */
```
Comment (camekman) [inline code/style.css:58]: Great way of explaining the code. Thank you!

File: code/style.css [hunk 14]
```diff
@@ -1,7 +1,347 @@
...
+  justify-content: center;
+}
+
+#hero video {
+  width: 100%;
+  height: 1000px;
+  position: absolute;
+  top: 0;
+  left:0;
+  z-index: 5;
```
Comment (camekman) [inline code/style.css:23]: Why do we use z-index? I am not familiar with it but will do some research :)

**Review by camekman — COMMENTED** [review]
You have created a great business site that full-fills all the requirements of the project, great job! I really enjoyed the styling of the page that went so well with your theme. The site works well in mobile, tablet and desktop view and it looks great with your hero video. Your code has a nice flow with a good structure which makes it easy to understand. I really like that you have connected your git hub page to your footer, a very nice touch. I am very impressed by the detail level, that you have even changed the colors for your radio buttons. That is really something you should be extra proud of!

File: code/index.html (hunk 2 above)
Comment (JuliaNiki) [reply code/index.html:20]: Exactly, thank you :)

File: code/style.css (hunk 3 above)
Comment (JuliaNiki) [reply code/style.css:29]: Nope, no reason, just the lazy way to ensure that the element is always on top :)

File: code/style.css (hunk 6 above)
Comment (JuliaNiki) [reply code/style.css:75]: thank youuu!

File: code/style.css (hunk 8 above)
Comment (JuliaNiki) [reply code/style.css:234]: I am actually not sure, will try it when I have time, but I wanted to secure that JS knows where to take the class from :)

File: code/style.css (hunk 14 above)
Comment (JuliaNiki) [reply code/style.css:23]: To secure that this element will be on top of the other that has a smaller z-index :) 
Yes, check it out, very useful thing https://www.w3schools.com/cssref/pr_pos_z-index.asp

File: code/index.html (hunk 11 above)
Comment (JuliaNiki) [reply code/index.html:160]: Thank you so much, feel even more proud after I've read your comments :) 
They made me realise again how much progress has been done.

**Pull request closed without merging**